docs/concepts/isis-fundamentals/label-dictionary.md

```diff
@@ -0,0 +1,261 @@
+# ISIS Label Dictionary
+
+## Introduction
+
+This document is a reference to the various objects and groups, which can be found in ISIS labels. The definition of each object/group, the parameter meanings, and an example will be given.
+
+## IsisCube Object
+
+The IsisCube Object is simply a container. It is used to house the various elements (groups and objects) that make up an ISIS cube. The IsisCube object is comprised of the Core object, which defines the cube size and pixel attributes and many optional groups that are described elsewhere in this document including Archive, BandBin, Instrument, Kernels, and Mapping Groups. While we present a whole IsisCube object we will save of the individual elements in their own sections.
+
+```
+Object = IsisCube
+  Object = Core
+    ^Core       = filename.dat
+    StartByte   = 65537
+    Format      = Tile
+    TileSamples = 128
+    TileLines   = 128
+
+    Group = Dimensions
+      Samples = 320
+      Lines   = 5392
+      Bands   = 1
+    EndGroup
+
+    Group = Pixels
+      Type       = Real
+      ByteOrder  = Lsb
+      Base       = 0.0
+      Multiplier = 1.0
+    EndGroup
+  EndObject
+
+  Group = BandBin
+    OriginalBand = 10
+    Center       = 14.88 <microns>
+    Width        = 0.87 <microns>
+    FilterNumber = 10
+  EndGroup
+EndObject
+```
+
+## Core Object
+
+The core object is used to house the groups (Dimensions and Pixels) that describe the size and pixel attributes of the cube, as well as the storage type of the data.
+
+```
+Object = Core
+  ^Core       = filename.dat
+  StartByte   = 65537
+  Format      = Tile
+  TileSamples = 128
+  TileLines   = 128
+
+  Group = Dimensions
+    Samples = 320
+    Lines   = 5392
+    Bands   = 1
+  EndGroup
+
+  Group = Pixels
+    Type       = Real
+    ByteOrder  = Lsb
+    Base       = 0.0
+    Multiplier = 1.0
+  EndGroup
+EndObject
+```
+
+|Keyword|Type|Description|
+|--- |--- |--- |
+|^Core|String|This optional keyword is used to point to an extra binary file containing the cube data. The cube is designated detached (labels and cube data in separate files) if this keyword exists.  If the keyword does not exist it indicates the labels and cube data are attached (in the same file).|
```

Comment:
Is the "^" part of the keyword, or is it part of the syntax?  I don't actually know the answer here, but I was under the impression that keywords prefixed with "^" meant that the data / file exists external to the cube.

Reply:
This is the original docs and it has `^Core` in the table. https://isis.astrogeology.usgs.gov/8.1.0/documents/LabelDictionary/LabelDictionary.html